.github/workflows/enclave-cc-cicd.yaml

```diff
@@ -0,0 +1,59 @@
+name: CI/CD | enclave-cc e2e tests
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator enclave-cc e2e tests
+    runs-on: ubuntu-22.04
+    services:
```

Comment:
When I created the e2e tests for the operator I used an external registry but It seems I didn't need that.

On confidential-containers/cloud-api-adaptor, there is a CI for a webhook which leverages `Kind` as well. There, we are pushing the controller image to the cluster with `kind load` command (see: https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster). There is one details that made me getting stalled until @bpradipt came to rescue me: the controller image *must* be tagged and the tag *must not* be *latest*.

Reply:
right. `kind load` makes the image available but the `ImagePullPolicy` overrides that local image. That's why a local registry works nicely.

Reply:
yep, the image must be tagged and not `latest` so that the `IfNotPresent` is set by default, as explained in https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster